.travis.d/deploy_docs.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+cd $TRAVIS_BUILD_DIR
+git fetch origin gh-pages:origin/gh-pages
+git checkout origin/gh-pages -b gh-pages || git checkout --orphan gh-pages
```

Comment:
at the develop this script, actually not pushed gh-pages in repo, and i want to update or create.

well.... yeah currently it's useless and can broken in that case
first time i made `fetch && checkout -b || git checkout --orphan`
it changed after fixing, so i'll return original cases..
but IMO, still need handling not exist gh-pages case.
if break at deploy time, travis will say `build fail`